src/main/java/duke/Task.java

```diff
@@ -0,0 +1,90 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * A general task class that handles the status of any task and methods to add a task to the task list.
+ */
+
+public class Task {
+    private String description;
+    private Boolean isDone;
+
+    /**
```

Comment:
Maybe include in the javadocs why you have 2 different constructors? Not necessary but would likely make for documentation that is easier to understand